### PR TITLE
Bugfix of RT calculation in MS mode.

### PR DESCRIPTION
**Pull request by uemoto1**
This pull request contains the bugfix claimed on the issue #232 .

### CODE REVIEW

File: src/ARTED/control/control_ms.f90 [hunk 1]
```diff
@@ -63,14 +63,12 @@ subroutine tddft_maxwell_ms
     end if
 
 !====RT calculation============================
-    if ( .not. (restart_option == 'restart')) then
```
Comment (shunsuke-sato) [inline src/ARTED/control/control_ms.f90:66]: Why do you remove this `if` statement? I added it to avoid incorrect initialization in PR #212.

**Review by shunsuke-sato — COMMENTED** [review]
@uemoto1 san, thank you for the implementation. I have a question. Please see below.

File: src/ARTED/control/control_ms.f90 (hunk 1 above)
Comment (uemoto1) [reply src/ARTED/control/control_ms.f90]: @shunsuke-sato san, it is not directly affect to this bugfix. Which bug tracking, I have exhaustively tested  if statements related to restarting. Then, I found Line:66 is doubly checks same condition defined in Line:47. 

It has not the origin of Issue #237, therefore I think both existing and removing are acceptable in this code.

**Comment by uemoto1** [comment]
@shunsuke-sato san, it is not directly affect to this bugfix. While bug tracking, I have exhaustively tested if statements related to restarting. Then, I found Line:66 is doubly checks same condition defined in Line:47.

It has not the origin of Issue #232, therefore I think both existing and removing are acceptable in this code.

**Comment by uemoto1** [comment]
@ayamada224 san, Thank you for merging.
I have struggled at the printf debugging, and I found something is strange in `read_external_input_for_restart`. In the case `input_for_restart` is not exist, only the root process skips the reading and broadcasting, but any other processes can't skip the broadcasts. I think this mismatch leads the desync of MPIs...

**Comment by uemoto1** [comment]
In this time, I have self-merge this PR, due to catch up the deadline of the next release...

**Pull request merged**